src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs

```diff
@@ -10,12 +10,14 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
-using System.Linq;
```

Comment:
@dotnet/roslyn-compiler There are trivial changes to only a handful for compiler files here - these files are linked into both Compilers and Workspaces, hence cannot be avoided in this PR.

Reply:
@dotnet/roslyn-compiler - can someone please take a quick look? Change just removes unused usings in few files.

Reply:
Looking

---
In reply to: [405789406](https://github.com/dotnet/roslyn/pull/43166#discussion_r405789406) [](ancestors = 405789406)

Reply:
Signed off.
@mavasani Is the plan to enable this enforcement on the entire roslyn repo down the line?

Reply:
> Is the plan to enable this enforcement on the entire roslyn repo down the line?

Given these are code style rules, we understand that each team might have different preferences on the settings as well as whether or not to enforce a specific rule. We are planning to dogfood a few code style rules in IDE layer for couple of weeks, at which point we will decide which rules to retain in CI, and which ones to just keep for IDE experience. We can then follow-up with the compiler team to see if they are interested in enforcement of any such rules in CI, and if so, we can work towards enabling them in the entire repo.